CHANGELOG.rst

```diff
@@ -8,10 +8,11 @@ All notable changes to the kytos project will be documented in this file.
 Added
 =====
 - '/kytos/configuration/' endpoint to display kytos configuration.
-- Add 'api_port' in kytos.conf. The default value is 8181.
+- Added 'api_port' in kytos.conf, with default value equals 8181.
```

Comment:
There is no need to write `Added` on the `Added` section....

Change this to:
``'api_port' in kytos.conf file, with '8181' as default value.``